app/views/hyrax/base/_form_metadata.html.erb

```diff
@@ -1,13 +1,16 @@
+
 <%= render 'form_media', f: f %>
 
 <% if @curation_concern.class.to_s == 'Audio' || @curation_concern.class.to_s == 'Video' %>
-  <div class="well transcript-selection">
-    <%= render_edit_field_partial(:transcript_id, f: f) %>
-  </div>
+    <div class="well transcript-selection">
+        <%= render_edit_field_partial(:transcript_id, f: f) %>
+    </div>
 <% end %>
 
 <div class='base-terms'>
-<%= render_edit_field_partial(:title,  f: f) %>
+<% unless current_page?(:controller => 'batch_uploads', :action => 'new') %>
```

Comment:
This doesn't work in the existing view specs because (I may not be phrasing this correctly) the controller isn't available in that kind of test.

Reply:
It looks like `rspec/rails` folks recommend stubbing the helper: https://github.com/rspec/rspec-rails/issues/1347

Could we do that and keep the tests? It seems like a waste to have spent the time writing them and then just delete them because there's churn in the view.

Reply:
The issue seems to be that this is a partial test: https://relishapp.com/rspec/rspec-rails/docs/view-specs/view-spec-infers-controller-path-and-action#do-not-infer-action-in-a-partial

Reply:
Hmm should I move them into a different folder?